### PR TITLE
[C3] add wrangler.toml binding examples to templates

### PR DESCRIPTION
**Pull request by RamIdeas**

### CODE REVIEW

File: packages/create-cloudflare/templates/common/js/wrangler.toml [hunk 1]
```diff
@@ -2,6 +2,39 @@ name = "<TBD>"
...
+# zone_id = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+
+# [env.production]
+# MY_VARIABLE = "production_value"
+
+# [env.staging]
+# MY_VARIABLE = "staging_value"
+
+# [env.shared]
+# SHARED_VARIABLE = "shared_value"
```
Comment (JacobMGEvans) [inline packages/create-cloudflare/templates/common/js/wrangler.toml:40]:
```suggestion
# SHARED_VARIABLE = "shared_value"

```

File: packages/create-cloudflare/templates/common/ts/wrangler.toml [hunk 2]
```diff
@@ -2,6 +2,39 @@ name = "<TBD>"
...
+# zone_id = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+
+# [env.production]
+# MY_VARIABLE = "production_value"
+
+# [env.staging]
+# MY_VARIABLE = "staging_value"
+
+# [env.shared]
+# SHARED_VARIABLE = "shared_value"
```
Comment (JacobMGEvans) [inline packages/create-cloudflare/templates/common/ts/wrangler.toml:40]:
```suggestion
# SHARED_VARIABLE = "shared_value"

```

File: packages/create-cloudflare/templates/simple/js/wrangler.toml [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+# zone_id = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+
+# [env.production]
+# MY_VARIABLE = "production_value"
+
+# [env.staging]
+# MY_VARIABLE = "staging_value"
+
+# [env.shared]
+# SHARED_VARIABLE = "shared_value"
```
Comment (JacobMGEvans) [inline packages/create-cloudflare/templates/simple/js/wrangler.toml:40]:
```suggestion
# SHARED_VARIABLE = "shared_value"

```

File: packages/create-cloudflare/templates/simple/ts/wrangler.toml [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+# zone_id = "zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz"
+
+# [env.production]
+# MY_VARIABLE = "production_value"
+
+# [env.staging]
+# MY_VARIABLE = "staging_value"
+
+# [env.shared]
+# SHARED_VARIABLE = "shared_value"
```
Comment (JacobMGEvans) [inline packages/create-cloudflare/templates/simple/ts/wrangler.toml:40]:
```suggestion
# SHARED_VARIABLE = "shared_value"

```

**Review by JacobMGEvans — CHANGES_REQUESTED** [review]
EOF extra line, formatter seems disabled or didn't work on the TOML for some reason.

**Review by jculvey — APPROVED** [review]

**Review by JacobMGEvans — APPROVED** [review]

**Pull request merged**